docs/getting-started.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Getting Started
+nav: 0.1
+---
+Welcome to the Jotai Getting Started tutorial!
+This tutorial will introduce you to Jotai and teach you how to start using it correctly.
+
+For this tutorial, we assume that you're using Jotai with React. The example is based on [a Create-React-App folder structure](https://create-react-app.dev/docs/folder-structure) so the first thing you have to do is:
+
+`npx create-react-app my-app`
+
+## Installation
+To install the latest stable version of Jotai, run the following command:
+
+`npm install jotai` or `yarn add jotai`
+
+## Provider
```

Comment:
Jotai Provider is optional, and for getting started, we should teach the provider-less mode.

Reply:
I was thinking about that too.
do u think I should remove the Provider but also mention that u can add it too?

Reply:
I would definitely remove it from the getting started example. That's one of the major reasons I developed the provider-less mode.
We can mention about Provider, and it would be nice if we can mention "when" we should use Provider. (I wonder if it's in the scope of getting started, or out of it.)

Reply:
I think it's best to only introduce concepts as they became necessary. The provider is not necessary for the getting started example so I think that section should be removed. 🙂